sdk/attestation/azure-security-attestation/samples/attestation/attest_openenclave_shared.cpp

```diff
@@ -6,18 +6,10 @@
  * for C++ to create, get, update, delete and purge a certificate.
  *
  * @remark The following environment variables must be set before running the sample.
- * - ATTESTATION_AAD_URL:  Points to an Attestation Service Instance in AAD mode.
- * - ATTESTATION_ISOLATED_URL:  Points to an Attestation Service Instance in Isolated mode.
- * - LOCATION_SHORT_NAME:  Specifies the short name of an Azure region to use for shared mode
+ * - ATTESTATION_AAD_URL: Points to an Attestation Service Instance in AAD mode.
+ * - ATTESTATION_ISOLATED_URL: Points to an Attestation Service Instance in Isolated mode.
+ * - LOCATION_SHORT_NAME: Specifies the short name of an Azure region to use for shared mode
  * operations.
- * - AZURE_TENANT_ID:     Tenant ID for the Azure account. Used for authenticated calls to the
- * attestation service.
- * - AZURE_CLIENT_ID:     The Client ID to authenticate the request. Used for authenticated calls to
- * the attestation service.
```

Comment:
Should we still call these out in docs? For `EnvCred` to work, don't these still need to be set by the user?